Sources/SwiftSyntaxBuilderGeneration/Templates/ResultBuildersFile.swift

```diff
@@ -0,0 +1,387 @@
+        
+        TypealiasDecl(
+          leadingTrivia: [
+            .newlines(1),
+            .docLineComment("/// The type of the final returned result, which defaults to Component if"),
+            .newlines(1),
+            .docLineComment("/// buildFinalResult() is not provided."),
+            .newlines(1)
+          ],
+          modifiers: [TokenSyntax.public],
+          identifier: "FinalResult",
+          initializer: type.buildable)
+        
+        
+        //      public static func buildBlock(_ components: Component...) -> Component {
+        //        return components.flatMap { $0 }
+        //      }
+        FunctionDecl(
+          leadingTrivia: [
+            .newlines(1),
+            .docLineComment("/// Required by every result builder to build combined results from"),
+            .newlines(1),
+            .docLineComment("/// statement blocks."),
+            .newlines(1)],
+          modifiers: [TokenSyntax.public, TokenSyntax.static],
+          identifier: .identifier("buildBlock"),
+          signature: FunctionSignature(
+            input: ParameterClause {
+              FunctionParameter(
+                firstName: TokenSyntax.wildcard,
+                secondName: TokenSyntax.identifier("components"),
+                colon: TokenSyntax.colon,
+                type: "Component",
+                ellipsis: TokenSyntax.ellipsis)
+            },
+            output: "Component")) {
+              ReturnStmt(
+                expression: FunctionCallExpr(MemberAccessExpr(
+                  base: "components",
+                  name: "flatMap")) {
+                    //            SequenceExpr(elements: ExprList([
+                    //              IntegerLiteralExpr(digits: "n"),
+                    //              BinaryOperatorExpr("-"),
+                    //              IntegerLiteralExpr(2),
+                    //            ]))
+                  }
+              )
+            }
+        
+        FunctionDecl(
+          leadingTrivia: [
+            .newlines(1),
+            .docLineComment("/// If declared, provides contextual type information for statement"),
+            .newlines(1),
+            .docLineComment("/// expressions to translate them into partial results."),
+            .newlines(1)
+          ],
+          modifiers: [TokenSyntax.public, TokenSyntax.static],
+          identifier: .identifier("buildExpression"),
+          signature: FunctionSignature(
+            input: ParameterClause {
+              FunctionParameter(
+                firstName: TokenSyntax.wildcard,
+                secondName: TokenSyntax.identifier("expression"),
+                colon: TokenSyntax.colon,
+                type: "Expression")
+            },
+            output: "Component")) {
+              ReturnStmt(
+                expression: ArrayElement(
+                  expression: IdentifierExpr("expression")
```

Comment:
I believe we can omit `IdentifierExpr`, the string literal should suffice for the compiler to pick up on the right `ExpressibleAs` chain IIRC.